src/components/Modal/Modal.js

```diff
@@ -0,0 +1,88 @@
+import {
+  FavIcon,
+  Icon,
+  IconWrapper,
+  CloseModal,
+  PhotoFooter,
+  PhotoHeader,
+  PhotoImage,
+  Subtitle,
+  TextWrapper,
+  PhotoImageWrapper,
+  SliderContainer,
+  Title,
+  Avatar,
+  AvatarImg,
+  StyledSlider,
+} from './Modal.styles';
+import heartIcon from 'assets/icons/heart.svg';
+import starIcon from 'assets/icons/star.svg';
+import closeIcon from 'assets/icons/cross.svg';
+import MagicSliderDots from 'react-magic-slider-dots';
+import 'react-magic-slider-dots/dist/magic-dots.css';
+import moment from 'moment'
+import { Link } from 'react-router-dom';
+
+export default function Modal({photos,index, show, hideModal }) {
+  const showHideModal = show
+    ? 'SliderContainer display-block'
+    : 'SliderContainer display-none';
```

Comment:
`const showHideModal =  `SliderContainer ${show ? 'display-bloc' : ' display-none'}`;`

Reply:
can you do this in styled components directly by passing a prop?